rest-service/manager_rest/test/endpoints/test_deployments.py

```diff
@@ -1069,3 +1069,82 @@ def test_update_attributes_already_set(self):
         ))
         with self.assertRaisesRegex(CloudifyClientError, 'already set'):
             self.client.deployments.set_attributes('dep1', description='d2')
+
+    def test_create_deployment_with_default_schedules(self):
+        _, _, _, deployment = self.put_deployment(
+            blueprint_file_name='blueprint_with_default_schedules.yaml')
+
+        sched_ids = [sc['id'] for sc in self.client.execution_schedules.list()]
+        sc1_id = '{}_{}'.format(deployment.id, 'sc1')
+        sc2_id = '{}_{}'.format(deployment.id, 'sc2')
+        self.assertListEqual([sc1_id, sc2_id], sched_ids)
+
+        sc1 = self.client.execution_schedules.get(sc1_id)
+        self.assertEquals(sc1['rule']['frequency'], '1w')
+        self.assertEquals(len(sc1['all_next_occurrences']), 5)
+
+    def test_update_deployment_with_default_schedules(self):
+        _, _, _, deployment = self.put_deployment(
+            blueprint_file_name='blueprint_with_default_schedules.yaml')
+
+        sc1_id = '{}_{}'.format(deployment.id, 'sc1')
+        sc2_id = '{}_{}'.format(deployment.id, 'sc2')
+        sc3_id = '{}_{}'.format(deployment.id, 'sc3')
+
+        sched_ids = [sc['id'] for sc in self.client.execution_schedules.list()]
+        self.assertListEqual([sc1_id, sc2_id], sched_ids)
+        self.assertEquals(
```

Comment:
Why do you make this assertion here and not in the previous test?

Reply:
copy error :)